plugin.py

```diff
@@ -79,27 +144,191 @@ class TellRecord(Base):
+                    elif t['private'] is False:
+                        _pub_count += 1
+                        _relay_pub = True
+                        _pub_tells.append(self.registryValue("tellMessage").format(**_d))
+
+                # Relay public tells
+                if _relay_pub:
+                    _m = _public.format(**{
+                        'to': msg.nick,
+                        'pub_count': _pub_count,
+                        'plural': "s" if _pub_count > 1 else ''
+                    })
+                    irc.queueMsg(ircmsgs.notice(_channel, _m))
+
+                    for m in _pub_tells:
+                        irc.queueMsg(ircmsgs.notice(_channel, m))
+
+                # Relay private tells
+                if _relay_private:
+                    _m = _private.format(**{
+                        'to': msg.nick,
+                        'priv_count': _priv_count,
+                        'plural': "s" if _priv_count > 1 else ''
+                    })
+                    irc.queueMsg(ircmsgs.notice(msg.nick, _m))
+
+                    for m in _priv_tells:
+                        irc.queueMsg(ircmsgs.notice(msg.nick, m))
+
+        return msg
+
     def tell(self, irc, msg, args, now, nicks, message):
         """<user1,user2> <message>
     
         Saves a tell for the specified nicks.
         """
-        
-        # TODO:
-        # Channel or private message to bot? Use wrap for the channel? Use ChannelDb?
-        
-        # Insert a Tell in the tell table
-        new_tell = TellRecord(FromNick=msg.nick, ToNick="Ownix", Content=message, Private=0, Read=0, Timestamp=datetime.datetime.now())
-        session.add(new_tell)
+        tell_to = nicks.split(',')
+
+        # Insert tell records per each nick name
+        _dt = datetime.datetime.now()
+        for i in tell_to:
+            new_tell = TellRecord(FromNick=msg.nick, ToNick=i, Content=message, Private=self.pm, Read=0,
+                                  Timestamp=_dt)
+            session.add(new_tell)
+
         session.commit()
-        
-        irc.reply(str("Saving tell '" + message + "' for " + nicks))
+
+        irc.queueMsg(ircmsgs.notice(msg.nick, "Saving tell '" + message + "' for " + nicks))
     tell = wrap(tell, ['now', 'somethingWithoutSpaces', 'text'])
 
+    def skiptells(self, irc, msg, args):
+        """
+        Skip all tells and set to Read.
+        """
+        self.queryTell.flag_all_read(msg.nick)
+
+        irc.queueMsg(ircmsgs.notice(msg.nick, "Beep boop beep. Skipping all tells."))
+
+    skiptells = wrap(skiptells, [])
+
+    def tellrefresh(self, irc, msg, args):
+        """
+        Refresh the unread tells from Database
+        """
+
+        self.queryTell.load_unread()
+
+        irc.queueMsg(ircmsgs.notice(msg.nick, "Beep boop beep. Tells have been reloaded from the database."))
+
+        return True
+
+    tellrefresh = wrap(tellrefresh, ['admin'])
+
+    def delay_tells(self, irc, msg, args, time):
```

Comment:
This function has the same name as a function in PostTell class. You should consider changing the library one to `set_delay` or something.

> There are only two hard things in Computer Science: cache invalidation and naming things.
> -- Phil Karlton